java/Pipfile

```diff
@@ -0,0 +1,26 @@
+[[source]]
```

Comment:
We don't need this Pipfile for Java, do we?

Reply:
It will show code java, but it is not the real Java application, isn't? I think python is nice to do it.